spec/acceptance/schema_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper_acceptance'
+
+describe 'openldap::server::schema' do
+
+  context 'without parameters' do
+    it 'creates an overlay' do
+      pp = <<-EOS
+      class { 'openldap::server': }
+      ->
+      openldap::server::schema { 'core':
+        ensure => present,
+      }
+      EOS
+
+      apply_manifest(pp, :catch_failures => true)
+      #apply_manifest(pp, :catch_changes => true)
```

Comment:
Ouch I didn't notice that I commented to second test on the overlay_spec.rb, I'll fix that.